### PR TITLE
Adding blog post for 2014 Review

### PR DESCRIPTION
**Pull request by aterrel**

### CODE REVIEW

File: content/articles/News/2015-02-07-reviewing-2014.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+=========================================
+:date: 2015-02-07
+:author: Andy R. Terrel
+:status: draft
+
+.. image:: /media/img/articles/2014-Mosaic.png
+    :height: 300px
+    :alt: 2014 Mosaic
+
+While not represented in our blog activity, 2014 turned out to be a very busy year for NumFOCUS. We sponsored more projects, new board members, and added programs.  It has been hard to keep up with all the activity within the organization, but perhaps much harder for those outside.  Let me go through a few of the highlights from the year to get folks caught up.
```
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: "gained new board members"?

File: content/articles/News/2015-02-07-reviewing-2014.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+First and foremost, we had the first election process. This has allowed us to welcome Lorena Barba, Brian Granger, Stefan Karpinski and Cindee Madison to the board of directors. In an effort to grow the number of people helping out we have also formed two new committees: Advisor Committee and Vice Presidents. The inaugural advisor committee consists of Fernando Perez and Travis Oliphant. They will help guide the organization as it grows in both scope and operations. The Vice President committee consists of Sheila Miguez, James Powell and Ben Zaitlin, all committed community members who have helped in various ways to get us started. We are always looking for more volunteers to help build our organization, but with these added team members we hope to continue to deliver high quality services to our community.
```
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: "Board of Advisers"

Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: and VPs aren't really a "committee" either, more like new roles.

File: content/articles/News/2015-02-07-reviewing-2014.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Over the year we held four PyData conferences, supported one fellowship, lead women in tech bootcamps, helped numerous conference attendees travel to important conferences and more. As the new year starts, we will be doing many of these things plus working with the `Training Up`_ program. We see the need to not only sponsor projects but to create important programs that get the community together. As always, we are always looking for more volunteers for these events, so please do get in touch.
```
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: "Women in Technology"

Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: "numerous" can we give an estimate? My guess would be 20-30.

Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: 2x always

**Comment by rgommers** [comment]
It would be helpful to break lines at 80 chars, both because it helps diffs and is the normal thing to do for reST files, but also because then I can actually put review comments somewhere near the text I'm commenting on.

File: content/articles/News/2015-02-07-reviewing-2014.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
+=========================================
+Reviewing 2014
+=========================================
+:date: 2015-02-07
+:author: Andy R. Terrel
+:status: draft
```
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: you don't need this "draft" label right?

File: content/articles/News/2015-02-07-reviewing-2014.rst [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Over the year we held four PyData conferences, supported one fellowship, lead women in tech bootcamps, helped numerous conference attendees travel to important conferences and more. As the new year starts, we will be doing many of these things plus working with the `Training Up`_ program. We see the need to not only sponsor projects but to create important programs that get the community together. As always, we are always looking for more volunteers for these events, so please do get in touch.
+
+Perhaps one of the most exciting things that we did this year is expand our language scope from a Python centric organization to include several non-Python projects. We have ROpenSci and Julia now a part of our line up. Joining us on the education front is Software Carpentry and sister organization Data Carpentry.  While fiscal sponsorship doesn’t make sense for every project, it is a service we happily provide to help projects sustain and grow in natural ways.
```
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: Make all these project names into links?

Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: typo: education front is --> are

File: content/articles/News/2015-02-07-reviewing-2014.rst (hunk 3 above)
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: PyData as a link?

File: content/articles/News/2015-02-07-reviewing-2014.rst [hunk 6]
```diff
@@ -0,0 +1,22 @@
+=========================================
+Reviewing 2014
+=========================================
+:date: 2015-02-07
+:author: Andy R. Terrel
+:status: draft
+
+.. image:: /media/img/articles/2014-Mosaic.png
+    :height: 300px
+    :alt: 2014 Mosaic
```
Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: I assume you didn't leave Brian out of this graphic on purpose?

Comment (rgommers) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: And AstroPy, SymPy and IPython also signed FSAs in 2014. Worth a mention, if not inclusion in the graphic.

**Comment by rgommers** [comment]
Nice summary Andy, thanks!

File: content/articles/News/2015-02-07-reviewing-2014.rst (hunk 6 above)
Comment (lsilen) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: Is it Advisory or Advisor Committee or Council

File: content/articles/News/2015-02-07-reviewing-2014.rst (hunk 2 above)
Comment (aterrel) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: Yeah we are trying ot avoid the word Board as much as possible. Unfortunately, in US corporate governance that comes with more liabilities than we want to put there.

File: content/articles/News/2015-02-07-reviewing-2014.rst (hunk 1 above)
Comment (aterrel) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: Thanks!

File: content/articles/News/2015-02-07-reviewing-2014.rst (hunk 4 above)
Comment (aterrel) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: The draft label is to post it on the site but not in the archives and link pages. So yeah its good until I get more eyes on the post.

File: content/articles/News/2015-02-07-reviewing-2014.rst (hunk 6 above)
Comment (aterrel) [inline content/articles/News/2015-02-07-reviewing-2014.rst]: Opps putting Brian in. Did everyone sign the FSA this year? I thought those projects did in 2013...

File: content/articles/News/2015-02-09-numfocus-2014-review.rst [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+While not represented in our blog activity, 2014 turned out to be a very busy year for NumFOCUS. We sponsored more projects, gained new board members, and added programs.  It has been hard to keep up with all the activity within the organization, but perhaps much harder for those outside.  Let me go through a few of the highlights from the year to get folks caught up.
+
+First and foremost, we had the first election process. This has allowed us to welcome Lorena Barba, Brian Granger, Stefan Karpinski and Cindee Madison to the board of directors. In an effort to grow the number of people helping out we have also formed two new ways for people to contribute: Advisor Council and Vice President roles. The inaugural advisor council consists of Fernando Perez and Travis Oliphant. They will help guide the organization as it grows in both scope and operations. The new Vice President are Sheila Miguez, James Powell and Ben Zaitlin, all committed community members who have helped in various ways to get us started. We are always looking for more volunteers to help build our organization, but with these added team members we hope to continue to deliver high quality services to our community.
```
Comment (quasiben) [inline content/articles/News/2015-02-09-numfocus-2014-review.rst]: `Ben Zaitlin` -> `Ben Zaitlen`

Comment (labarba) [inline content/articles/News/2015-02-09-numfocus-2014-review.rst]: Small typos:
The new Vice President are … —> The new Vice Presidents are … (plural)

File: content/articles/News/2015-02-09-numfocus-2014-review.rst [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+Over the year we held four `PyData conferences`_, supported one `John Hunter Technology Fellow`_, lead Women in Technology bootcamps, helped between 20 and 30 conference attendees travel to important conferences, and more. As the new year starts, we will be doing many of these things plus working with the `Training Up`_ program. We see the need to not only sponsor projects but to create important programs that get the community together. As always, we are always looking for more volunteers for these events, so `please do get in touch`_.
```
Comment (labarba) [inline content/articles/News/2015-02-09-numfocus-2014-review.rst]: lead Women in Technology bootcamps … —> led Women in Technology bootcamps … (past tense)

File: content/articles/News/2015-02-09-numfocus-2014-review.rst [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+Perhaps one of the most exciting things that we did this year is expand our language scope from a Python centric organization to include several non-Python projects. We have `ROpenSci`_ and `Julia`_ now a part of our line up. Joining us on the education front are `Software Carpentry`_ and sister organization `Data Carpentry`_.  While fiscal sponsorship doesn’t make sense for every project, it is a service we happily provide to help projects sustain and grow in natural ways.
+
+In the coming year, we plan to be a great deal more vocal about our activities. Personally, two major goals I have is to expand our reach to Europe and start making headway on project governance recommendations. But that is another story.
```
Comment (labarba) [inline content/articles/News/2015-02-09-numfocus-2014-review.rst]: two major goals I have is … —> two major goals I have are …

**Pull request merged**